### ISSUE TITLE
Wrong Banner height in Landscape Mode

### ISSUE THREAD

**Issue by RonnySchleicher** [floatinghotpot/cordova-plugin-admob#221]
#### Testsystems
Samsung S3 (GT-I9300) with Android 4.3
Motorola X (2. Gen.) with Android 6.0

#### Steps to Reproduce
Create a new fresh Phonegap App with follow Script (in this case on a Windows 10 PC).
I use the newest Version of Cordova in this case.

```
call npm uninstall -g cordova
call npm install -g cordova

call cordova create PhoneGap_android de.bannertest.bannertest Bannertest

cd PhoneGap_android

call cordova platform add android

call cordova plugin add cordova-plugin-whitelist@1.0
call cordova plugin add cordova-plugin-device-orientation
call cordova plugin add cordova-plugin-device
call cordova plugin add cordova-plugin-console
call cordova plugin add cordova-plugin-admob
```

Open the new Project with the Android Studio 2.2 and install it on your Android Phone.

In Landscape Mode is the Bannersize okay.

![image](https://cloud.githubusercontent.com/assets/1914203/18850610/971a13a0-8438-11e6-9772-f46c59321aca.png)

But in the Landscape Mode the Plugin shows the same Banner.

![image](https://cloud.githubusercontent.com/assets/1914203/18850670/caf9a76c-8438-11e6-953a-1dd959d14df2.png)

Normally the banner in landscape mode should fill the entire width and not be so high. Just as the red outlined area. 

![image](https://cloud.githubusercontent.com/assets/1914203/18850765/3a3674b6-8439-11e6-8fd8-04ca534bc205.png)

The Banner height does not match the Google documentation.

https://firebase.google.com/docs/admob/ios/banner

**Comment by MBuchalik** [comment]
Does it show the correct banner when starting the application in landscape mode? I think you need to re-create the banner on orientationchange.

**Comment by RonnySchleicher** [comment]
### You asked me two questions
#### Does it show the correct banner when starting the application in landscape mode?
Yes. When I start in landscape mode, the banner will be displayed correctly.
But if I then switch back to portrait mode no banner is displayed at all.

To make it clear I have here set 
```
overlap       : false
```
in this Screenshot.

![image](https://cloud.githubusercontent.com/assets/1914203/18880130/28a1edc2-84d7-11e6-9e28-51dbd8f9828b.png)


#### What do you What do you mean exactly with Re-create?

**Comment by MBuchalik** [comment]
Correct - at each orientation change, I call window.plugins.AdMob.destroyBannerView(); and window.plugins.AdMob.createBannerView({...}); That's what I meant with "re-create".

Maybe, you need to do this after a short timeout - just do some experiments. In my application, I re-create the banner 100ms after the orientationchange event happened. But this could just be a detail needed in my app and not in yours.

I am still wondering why this isn't part of the native code...

**Comment by floatinghotpot** [comment]
Yes, as the banner size is different when portrait or landscape mode, you need detect the orientation change and re-create the banner.

This is handled automatically in native code if you use the pro version.

**Comment by RonnySchleicher** [comment]
Hello Raymond Xie.

I wonder why the automatic switching is supported for iOS and not for Android?!

The idea of the same JavaScript code for different operating systems is completely bypassed in this case. I have to write for iOS other JavaScript code than for Andriod. Although I used the same plugin for both systems.

In my opinion, this behavior is a bug in cordova-plugin-admob.
